scripts/jest/TestFlags.js

```diff
@@ -73,6 +73,10 @@ function getTestFlags() {
       classic: releaseChannel === 'classic',
       www,
 
+      // Using this more specific flag so it's easier to clean up later
+      enableLegacyHiddenType:
+        featureFlags.enableNewReconciler === false || __EXPERIMENTAL__,
```

Comment:
Name is a bit misleading, but what this says is that the tests will pass if we're in the experimental branch of the new fork (because `unstable_LegacyHidden` is exposed) or if we're in the old fork (because the tests fall back to `<div hidden />` in the old fork).